db/migrate/20150520144034_add_coupon_to_member.rb

```diff
@@ -0,0 +1,5 @@
+class AddCouponToMember < ActiveRecord::Migration
+  def change
+    add_column :members, :coupon, :string
```

Comment:
Good point. I'm using sqlite this end, so that might have something to do with it?